### DISCUSSION TITLE
Is shopware open source ?

### DISCUSSION THREAD

**Discussion by khanakia** [shopware/platform#3189] [RFC / Proposal]
I am a little confused is shopware open source if yes then what are these pricing for here https://www.shopware.com/en/pricing/?

Will someone be able to have some insight on this please?

**Comment by keulinho** [comment]
Yes shopware is open source, but not all of the products/features are.

There is the [shopware community edition](https://www.shopware.com/en/community/community-edition/) that is 100% MIT licensed and thus open source. That's what is included in this repository.

But there are additional commercial plans available and with those plans you get access to more advanced features that are not open source, but are under a proprietary license. But all those features are add-ons, the core e-commerce solution is 100% open source.
